.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: bash
 
 env:
-  - MAVEN=3.6.3 JDK=8 ARGS=-Dair.java.version=1.8
-  - MAVEN=3.6.3 JDK=11 ARGS=
-  - MAVEN=3.6.3 JDK=14 ARGS=
+  - IMAGE=3.6.3-adoptopenjdk-8 ARGS=-Dair.java.version=1.8
+  - IMAGE=3.6.3-adoptopenjdk-11 ARGS=
+  - IMAGE=3.6.3-adoptopenjdk-15 ARGS=
+  - IMAGE=3.6.3-adoptopenjdk-16 ARGS=
```

Comment:
that does not exist. Maybe use `maven:3.6.3-openjdk-16` instead?

Reply:
Thanks, fixed